### DISCUSSION TITLE
How do I find all the sandboxes I created while being in Anonymous mode?

### DISCUSSION THREAD

**Discussion by mxmlnglt** [codesandbox/codesandbox-client#4347] [General]
Currently using Codesandbox.io to trial test some React libraries/components, now I can't fork anyone of them because of this error message:

We were unable to fork the sandbox
You need to sign in to create more than 3 sandboxes

I still had a tab in Chrome with one Codesandbox I didn't want to use anymore, so I find how to & deleted it, but now I still get the error.

In your T&C's you say: https://codesandbox.io/legal/privacy
> Cookies. CodeSandbox uses tracking, functional and analytical cookies because it enables many convenient features that may save you some time. We use tracking cookies to gather a unique view count of sandboxes and to match a sandbox to users without an account.

Following this path, is there a way to identify the URL's of the remaining sandboxes to be able to delete them?

Thanks...

PS: I don't want/have the time to create an account.

**Comment by mxmlnglt** [comment]
Well, I unfortunately clicked on something and your website automatically logged me in with my GitHub account (without asking me any confirmation... o.O) - plus I don't remember to have ever use Codesandbox.io WITH my GitHub account.

Nevermind, I found all 4 (!!!) sandboxes that seemed to have been created from the same original sandbox, https://codesandbox.io/s/wy2rl1nyzl

But I still think either I missed something, or there's something missing with the Anonymous mode...

**Comment by garethx** [comment]
Glad you found the projects, we can help you remove them if you're unable to.

To clarify about the 'without asking me any confirmation' bit. That's because you already created an account via GitHub on December 12, 2019, so you'd already been through the necessary confirmations.
